Source/PluginProcessor.cpp

```diff
@@ -77,7 +82,7 @@ void SoomplerAudioProcessor::setCurrentProgram (int index)
 
 const String SoomplerAudioProcessor::getProgramName (int index)
 {
-    return "Soompler";
+    return {};

```

Comment:
Why empty?

Reply:
No programs for this plugin yet. Program is preset with stored parameters values to be reproduced later.